frame/balances/storage.go

```diff
@@ -6,18 +6,19 @@ import (
 )
 
 var (
-	keyBalances      = []byte("Balances")
+	keyBalances = []byte("Balances")
+	//keyInactiveIssuance = []byte("InactiveIssuance")
 	keyTotalIssuance = []byte("TotalIssuance")
 )
 
 type storage struct {
 	TotalIssuance support.StorageValue[sc.U128]
-	//InactiveIssuance *support.StorageValue[sc.U128]
+	//InactiveIssuance support.StorageValue[sc.U128]
```

Comment:
Is there a reason to keep the commented parts?

Reply:
This is used in modules, which we do not have at the moment